custom_components/bdr_thermostat/BdrAPI.py

```diff
@@ -61,7 +61,8 @@ async def _store_token(self, token):
         self.token = token
 
     async def _login(self):
-        api_endpoint = self.endpoints["LOGIN"]
+        api_endpoint = f"https://remoteapp.bdrthermea.com/user/{self._brand}/login",
```

Comment:
Something also seems wrong here: `requests.exceptions.InvalidSchema: No connection adapters were found for "('https://remoteapp.bdrthermea.com/user/remeha/login',)"`.

Reverting to a fixed string with that same URL fixes that problem.

Reply:
That is strange it works fine for me. Currently travelling, could have a look in 2 weeks

Reply:
I have removed the , at the and of this line:
`api_endpoint = f"https://remoteapp.bdrthermea.com/user/{self._brand}/login"`

I have added the line:
vol.Required(CONF_PAIR_CODE): cv.string,

Restart HA and add integration

Via the eTwist app I have generated the pairing code: Home > Settings >  Connect devices and services
Invite someone, this will show a pairing ID

Reply:
Hey good that you found it. Could you make an PR including your fixes?

Reply:
Scratch that I made a PR for it, thanks for debugging: https://github.com/freitdav/BDRthermostatHA/pull/6